.github/pull_request_template.md

```diff
@@ -1,17 +1,20 @@
-## What is this PR doing?
+# What is this PR doing?
 
 
-## How can these changes be manually tested?
+# How can these changes be manually tested?
 
 
-## Does this PR resolve or contribute to any issues?
+# Does this PR resolve or contribute to any issues?
 
 
-## Checklist
+# Checklist
 - [ ] I have manually tested these changes
 - [ ] Post a link to the PR in the group chat
 
-## Guidelines
+# Guidelines
+- Please be careful when handling disagreements and stay mindful of others
 - If your PR is not ready, mark it as a draft
+- Prefer to seek followup instead of requiring changes, especially if the PR is already a clear net improvement
+- If appropriate, consider implementing followup changes yourself instead of waiting for them from the author and open a new PR against this one.
```

Comment:
I disagree on this point, 
doing the followup changes yourself does accelerate the process of merging the PR but the author is left without context about how their own work can be improved next time. The author can indeed go through the changes made afterwards but most of the times they would just let go of things (I am personally guilty of doing this). 

I don't think "rushing" into merging the PR is a good idea. 
This also resonates in this comment
> Prefer to seek followup instead of requiring changes, especially if the PR is already a clear net improvement

I would personally prefer if the PR is "complete in itself" for the feature it is adding rather than having multiple "follow ups" even if it requires a little extra time.

Reply:
Hmm. I'm trying to capture this with 'if appropriate', because it's difficult to define when it makes sense.

Having said that, are you suggesting that it's generally not a good idea? Definitely something worth chatting about, especially if you feel there are concrete examples where this has happened.